src/sourceMaps/sourceMap.ts

```diff
@@ -223,7 +224,7 @@ export class SourceMap {
             return null;
         } else {
             return {
-                line: position.line - 1, // Back to 0-indexed lines
+                line: global["evalSources"] && global["evalSources"].find(s => s == this._generatedPath) ? position.line : position.line - 1, // check if eval code else Back to 0-indexed lines
```

Comment:
Same as previous answer. I think either generated and authored code relay on`(function(){\n` problem.
in my opinion, it is more +1 indexed than 1-indexed: I keep the base index but add some adjustments.